tutor/resources/styles/components/task-plan/controls.less

```diff
@@ -8,6 +8,11 @@
     font-weight: 600;
     border-radius: 3px;
 
+    &.preview-btn {
+        border-radius: 4px;
+        font-weight: normal;
```

Comment:
hmmm the mock up has them bolded.  hmm nvm, dont mind me.  @nathanstitt, does this look good to you?  im ready to merge if you are.